packages/hash/api/src/graphql/typeDefs/block.typedef.ts

```diff
@@ -87,4 +87,16 @@ export const blockTypedef = gql`
     entity: UnknownEntity!
     componentId: ID!
   }
+
+  input BlockFilter {
+    accountId: ID!
+    entityId: ID!
+  }
+
+  extend type Query {
+    """
+    Get a specified list of blocks by accountId and entityId
+    """
+    blocks(blocks: [BlockFilter!]): [Block!]!
```

Comment:
Non-blocking
```suggestion
    blocks(blocks: [BlockFilter!]!): [Block!]!
```
Resolver returns an empty array when `blocks` is undefined. I assume we can just make it required :)

Reply:
Yup, good catch, thanks!